From 0b7e90f1ab5dbff4e6f68b67c57c340deab8a397 Mon Sep 17 00:00:00 2001
From: Bogdan Drozd <python.bogdan@gmail.com>
Date: Thu, 22 Sep 2022 02:04:13 +0300
Subject: [PATCH] a

---
 tree-sitter-langs-build.el | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tree-sitter-langs-build.el b/tree-sitter-langs-build.el
index 72280ae..3fbf04b 100644
--- a/tree-sitter-langs-build.el
+++ b/tree-sitter-langs-build.el
@@ -33,7 +33,7 @@
   "The directory where the library `tree-sitter-langs' is located.")
 
 ;; TODO: Separate build-time settings from run-time settings.
-(defcustom tree-sitter-langs-grammar-dir tree-sitter-langs--dir
+(defcustom tree-sitter-langs-grammar-dir "/usr/lib64/emacs-tree-sitter-langs-grammar/"
   "The root data directory of `tree-sitter-langs'.
 The 'bin' directory under this directory is used to stored grammar
 binaries (either downloaded, or compiled from source).
@@ -46,7 +46,7 @@ This should be set before the grammars are downloaded, e.g. before
 (defun tree-sitter-langs--bin-dir ()
   "Return the directory to stored grammar binaries.
 This used for both compilation and downloading."
-  (concat (file-name-as-directory tree-sitter-langs-grammar-dir) "bin/"))
+  (file-name-as-directory tree-sitter-langs-grammar-dir))
 
 ;; ---------------------------------------------------------------------------
 ;;; Utilities.
-- 
2.35.1

